invoke.ldap.yml

```diff
@@ -0,0 +1,10 @@
+---
+nautobot_docker_compose:
+  project_name: "nautobot-docker-compose"
+  python_ver: "3.9"
+  local: false
+  compose_dir: "environments/"
+  compose_files:
+    - "docker-compose.postgres.yml"
+    - "docker-compose.ldap.yml"
```

Comment:
Because the LDAP compose file is identical to base but uses the LDAP Dockerfile. I guess we could just use overrides in this case but was trying to make it easier to understand.